### ISSUE TITLE
"Related Articles" component should have anchors

### ISSUE THREAD

**Issue by rudygalfi** [ampproject/ampstart#210]
Since the use case for related articles is linking, I think it should have anchors in the code sample. Clicking either the image or the text should navigate.

Note that this would make it more consistent with "Related Articles Section", which already has an anchor tag defined in the sample.

**Comment by ericlindley-g** [comment]
Agreed — @spacedino, I'm thinking the entire div should link on click (rather than individual image and text). WDYT?

**Comment by spacedino** [comment]
yes, tap target should be entire container w image / text etc.
